src/pages/landingPage.js

```diff
@@ -19,37 +16,75 @@ import { Fragment } from 'react';
 
 
 function LandingPage() {
-  const start=()=>{
-    window.location.hash='/report-repairs'
-  }
+  const contactDetailsPageLink = (
+    <Link href="https://www.lincoln.gov.uk/contact-1/get-touch">
+      contact details page
+    </Link>
+  )
+
   return (
     <Fragment>
       <GridRow>
         <GridCol setWidth="two-thirds">
           <H1>Report a repair</H1>
```

Comment:
Title from [prototype](https://southwark-repairs-alpha-master.herokuapp.com/beta/v2/) is "Request a repair"
```suggestion
          <H1>Request a repair</H1>
```